articles/monitoring/guides/monitor-applications.md

```diff
@@ -13,20 +13,48 @@ useCase:
 
-Auth0 recommends using an authentication flow that doesn't require a user interface such as the **Resource Owner Password Grant**. That way, you can use a monitoring tool that doesn't have to mimick the actions of a user. Many monitoring services exist with this capability including:
+> There _might_ be an issue with Auth0, but the issue could also either be specific to the tenant used for the synthetic transaction or just a single failed request.
 
-* [New Relic](http://newrelic.com)
-* [Pingdom](http://pingdom.com)
+For synthetic transactions we recommend using setups that are close to your production tenant configuration as possible, potentially even using the same production tenant. Since setting up synthetic transactions with redirect flows and third party providers can be tricky, using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant) is recommended. This flow does neither involve browser redirects nor require a UI.
 
-Use one of these services to execute synthetic authentication requests. 
+If you are using [rules](/rules) or [custom database connection(s)](/connections/database/custom-db) or other extensibility points, the synthetic transaction(s) should be configured to go through their logic.
 
-## Keep reading
+Tools like [Pingdom](http://pingdom.com) make setting up synthetic transactions a simple thing.
+
+### Check Period
+We recommend synthetic transactions happen with a periodicity of one minute. For this simple monitoring approach, that's a frequency that will not consume a lot of your Auth0 rate limit quota, while also providing timely responses if an issue were to be happening.
+
+### Limitations of synthetic transactions
+Synthetic transactions do not represent your end user's experience, but instead aim to provide a proxy metric for them. They might not be exercising the same flows your users are, they lack atomicity (typically run once a minute) and do not report on errors your end users might have seen. However, they are a simple and inexpensive way of getting good signal on the health of an Auth0 tenant.
+
+If you are interested in getting more granular data read about [Error Tracking](#error-tracking) and [Metrics and Logs](#metrics-and-logs).
+
+## Error Tracking
+This approach is useful to track errors in existing calls to Auth0. It involves reporting errors whenever a call to Auth0 fails. [Sentry](https://sentry.io/) is a tool commonly used for these cases, which works both on frontend and backend scenarios.
+
+This approach is useful because it allows you to know about real errors that your end users are experiencing. However, because you are only tracking errors (and not all requests) it is not possible to get an accurate perception of "how many" end users are affected: is it 1% of 5%? It also doesn't require you to set up a separate "synthetic call", which might consume part of your rate limit quota, especially if misconfigured.
```

Comment:
```suggestion
This approach is useful because it allows you to know about real errors that your end users are experiencing. However, because you are only tracking errors (and not all requests) it is not possible to get an accurate perception of "how many" end users are affected: is it 1% or 5%? It also doesn't require you to set up a separate "synthetic call", which might consume part of your rate limit quota, especially if misconfigured.
```